Source/GMCAbilitySystem/Public/Components/GMCAbilityComponent.h

```diff
@@ -363,7 +363,7 @@ class GMCABILITYSYSTEM_API UGMC_AbilitySystemComponent : public UGameplayTasksCo
 private:
 
 	// return true if the ability is allowed to be activated considering active tags
-	bool CheckActivationTags(const UGMCAbility* Ability) const;
+	virtual bool CheckActivationTags(const UGMCAbility* Ability) const;
```

Comment:
you probably want to move this to the protected section too then :)

Reply:
Yep - updated, thanks!